main.py

```diff
@@ -481,6 +484,92 @@ def delete_interface_metadata(self, request: Request) -> JSONResponse:
         self.notify_metadata_changes(interface, 'removed')
         return JSONResponse("Operation successful")
 
+    @staticmethod
+    def _get_tag_type(content):
+        tag_type = content.get("tag_type", '1')
+        if tag_type != str(TAGType.VLAN.value):
+            detail = f"The TAG type {tag_type} is not allowed."
+            raise HTTPException(400, detail=detail)
+        return tag_type
+
+    @staticmethod
+    def check_non_range(tag_range):
```

Comment:
If you could rename it for what it actually does, it's mapping the data structure not only checking

Reply:
Changed to `mapp_singular_values`